src/Artsy/Router/Components/StorybookRelay.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react"
+import { QueryRenderer } from "react-relay"
+import { Environment, RecordSource, Store } from "relay-runtime"
+import { createMockNetworkLayer } from "../../Relay/createMockNetworkLayer"
+
+export const StorybookRelay = ({
```

Comment:
I like it 👌 

Is it really only for storybooks, though? Seems like this can be used for tests as well, in which case the name could do with updating too.